Dockerfile

```diff
@@ -17,9 +17,16 @@ RUN set -x && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
 
-ENV HADOOP_MAJOR_MINOR_VERSION 2.7
-ENV HADOOP_VERSION ${HADOOP_MAJOR_MINOR_VERSION}.3
-ENV SPARK_VERSION 2.0.1
+ENV HADOOP_MAJOR_MINOR_VERSION 2.6
+ENV HADOOP_VERSION ${HADOOP_MAJOR_MINOR_VERSION}.1
+
+ENV SPARK_16_VERSION 1.6.1
+ENV SPARK_20_VERSION 2.0.1
+
+######################################
+# Use Spark 1.6.1 as the default
+######################################
+ENV SPARK_VERSION ${SPARK_16_VERSION}
```

Comment:
Should switch this to `ARG` since this is a build time configuration.

Also, if set to `ENV`, might give the impression that the user can switch the version of spark at will. But this won't work because of the environment interpolation happening below.